docs/https_source_interface.rst

```diff
@@ -59,3 +59,12 @@ The webserver configuration will be updated to apply the HTTPS settings.
 Confirm that you can access the Source Interface at
 ``https://<your_domain>.onion``, and also that the HTTP URL
 ``http://<your_domain.onion`` redirects automatically to HTTPS.
+
+.. note:: By default, Tor Browser will send an OCSP request to a Certificate
+    Authority (CA) to check if the source interface certificate has been revoked.
+    Fortunately, this occurs through Tor. However, this means that a CA or anyone
+    along the path can learn the time that a Tor user visited the SecureDrop
+    source interface. In SecureDrop 0.4.1, OCSP stapling support will be added
```

Comment:
Nit: should be "Source Interface" rather than "source interface":

```
$ grep -r 'Source Interface'  docs/ | wc -l
32
$ grep -r 'source interface'  docs/ | wc -l
2
```

Note there are two occurrences of "source interface" in the proposed diff.